src/pages/Topic/Topic.tsx

```diff
@@ -112,46 +162,71 @@ export const Topic = ({ useTopic = _useTopic }: TopicProps): JSX.Element => {
     isGrouped
   ])
 
+  /**
+   * Update the learning path element status for the user after he closes a learning Element (iframe)
+   * @param user
+   */
+  const updateLearningPathElementStatus = async (user: User) => {
```

Comment:
async probably not needed
```suggestion
  const updateLearningPathElementStatus = (user: User) => {
```